src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/resource_allocation_manager_impl.cc

```diff
@@ -213,10 +215,103 @@ void ResourceAllocationManagerImpl::RemoveAppsSubscriptions(const Apps& apps) {
   }
 }
 
+template <typename EnumType>
+EnumType StringToEnum(const std::string& str) {
+  using smart_objects::EnumConversionHelper;
+  EnumType val;
+  EnumConversionHelper<EnumType>::StringToEnum(str, &val);
+  return val;
+}
+
+void ConstructOnRCStatusNotificationParams(
+    smart_objects::SmartObject& msg_params,
+    const std::map<std::string, uint32_t>& allocated_resources,
+    const std::vector<std::string>& supported_resources,
+    const u_int32_t app_id) {
```

Comment:
Shouldn't this be `uint32_t`?

Reply:
@JackLivio You're right. Fixed 4799472a4d34c1e947ba5b075d31923877295748